src/components/InventoryTabs/HybridInventoryTabs.js

```diff
@@ -0,0 +1,67 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import { useLocation } from 'react-router-dom';
+import { Tab, TabTitleText, Tabs } from '@patternfly/react-core';
+import useFeatureFlag from '../../Utilities/useFeatureFlag';
+import { resolveRelPath } from '../../Utilities/path';
+import useInsightsNavigate from '@redhat-cloud-services/frontend-components-utilities/useInsightsNavigate/useInsightsNavigate';
+import { manageEdgeInventoryUrlName } from '../../Utilities/edge';
+
+const HybridInventoryTabs = ({
+  ConventionalSystemsTab,
+  ImmutableDevicesTab,
+}) => {
+  const navigate = useInsightsNavigate();
```

Comment:
I think we better leave some TODO notes on removing the useInsightsNavigate usage: once we complete all of the router work, we have to migrate to useNavigate from react-router-dom.

Reply:
Hmm, should we keep useInsightsNavigate hook afterward also for a unified routing experience? I kinda like it now :)

Reply:
I would vote for keeping the `useInsightsNavigate` hook. We should improve it though.

There might be a way to accomplish the same with the raw react router hook or `Link` component, and if we do find out how to do it that way, we can deprecate our custom `useInsightsNavigate` and `InsightsLink` components.

Reply:
Yeah, this makes sense